xtest.sh

```diff
@@ -2,7 +2,7 @@
 total=0
 count=1
 
-limit_for_each=100
+limit_for_each=1000
```

Comment:
Ah... this is my bug of test framework. I'll merge it and would fix the bug. Thanks